src/components/daily/foodtooltip/FoodTooltip.js

```diff
@@ -36,7 +36,7 @@ class FoodTooltip extends PureComponent {
     const food = this.props.food;
     const rows = [
       <div key={'carb'} className={styles.carb}>
-        <div className={styles.label}>{t('Carbs')}</div>
+        <div className={styles.label}>{t('Rescuecarbs')}</div>
```

Comment:
To match the translation (without duplicate):
```suggestion
        <div className={styles.label}>{t('rescuecarbs')}</div>
```

Reply:
ok